pkg/static/rotater/lambda_function.py

```diff
@@ -1,18 +1,11 @@
-// Package static contains some static data, we could probably have
-// used embed here, but maybe some other time..
-package static
-
-// SecretsManagerPostgresRotationSingleUserBody contains the code for the AWS Secrets Manager rotation for postgres
-// - https://github.com/aws-samples/aws-secrets-manager-rotation-lambdas/blob/master/SecretsManagerRDSPostgreSQLRotationSingleUser
```

Comment:
Can we keep this comment, so we know where to get the latest content for this file?

Reply:
good point, will add it to the readme